cli_reference/differences_oc_kubectl.adoc

```diff
@@ -0,0 +1,40 @@
+[[cli-reference-differences-oc-kubectl]]
+= Differences Between OC and Kubectl
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+A copy of `kubectl` is included with the {product-title} command line interface (CLI).
+Although there are several similarities between these two clients, this guide will
+aim to clarify the main reasons and scenarios for using one over the other.
+
+== Using OC
+
+The `oc` binary offers full support for all of the features offered by an {product-title} cluster.
+It provides functionality not offered by `kubectl`, such as:
+
+- Full support for OpenShift resources (DeploymentConifgs, BuildConfigs, Routes, ImageStreams, ImageStreamTags, etc.).
+These are resources specific to {product-title}, and not native to Kubernetes.
+- Authentication: The `oc` binary offers a built-in `login` command which allows authentication
+into a cluster based on specified configuration. See
+xref:../dev_guide/authentication.adoc[developer authentication] and
+xref:../install_config/configuring_authentication.adoc[configuring authentication] for more information.
+- Additional commands, such as `new-app`, which make it easier to get new applications started using existing source code.
+
+== Using Kubectl
+
+The `kubectl` binary is provided as a means to support existing workflows and scripts for users coming from a DIY
+Kubernetes environment.
+
+Existing users of `kubectl` can continue to use this binary with no changes to the api, but should consider upgrading
+to `oc` in order to gain the added functionality mentioned in the previous section.
```

Comment:
The move from `kubectl` to `oc` should be additionally simple that `oc` is built on top of `kubectl` which means that all that is required is to change the binary name from `kubectl` to `oc`.